package.json

```
@@ -426,4 +426,4 @@
     "node": "20.19.4",
     "npm": "10.8.2"
   }
-}
+}
```

Please add back the trailing newline:


```suggestion
}

```